docs/agent-api.asciidoc

```diff
@@ -581,11 +584,13 @@ The HTTP request is otherwise ended at regular intervals controlled by the <<api
 [[server-timeout]]
 ===== `serverTimeout`
 
-* *Type:* Number
-* *Default:* `30`
+* *Type:* String
+* *Default:* `30s`
 * *Env:* `ELASTIC_APM_SERVER_TIMEOUT`
 
-Specify the response timeout in seconds when reporting transactions to APM Server.
+Specify the response timeout when reporting transactions to APM Server.
+The timeout should be in milliseconds,
```

Comment:
Previously the default unit was seconds. I think the other agents who have implemented this also use seconds as the default unit. Would it make sense to keep this default unit until v3.0.0?

Reply:
Well, we also have `abortedErrorThreshold`, which is currently counted in milliseconds, so I think we should at least unify them all on _something_.

Reply:
Updated to default to seconds.

Reply:
I do think that we should wait until the unit is required (v3.0.0) to make any changes to the default units.

Reply:
Yeah, I switched the default back to seconds, so it should be backwards compatible. 👍